units/meson-add-wants.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/sh -e
+
+unitdir="$1"
+target="$2"
+unit="$3"
+
+unitpath="${DESTDIR}${unitdir}/${unit}"
+dir="${DESTDIR}${unitdir}/${target}"
+
+if [[ "$target" == */ ]]; then
```

Comment:
Does the script file have +x mode enabled?

Does running `ninja reconfigure` before `ninja install` fix it?